Makefile

```diff
@@ -23,7 +39,7 @@ MD5SUM = md5sum
 TINYGO ?= tinygo
 
 # Use CCACHE for LLVM if possible
-ifneq (, $(shell which ccache))
+ifneq (, $(shell command -v ccache 2> /dev/null))
```

Comment:
What's the difference between `which` and `command -v`?

Reply:
The `command` is a shell builtin, whereas `which` is a command. Also adding the redirect solved the issue where it would print error messages if `ccache` is not installed.